adoc/deployment-bootstrap.adoc

```diff
@@ -798,3 +798,32 @@ kube-system   kured-zv696                             1/1       Running   0
 kube-system   oidc-dex-55fc689dc-b9bxw                1/1       Running   0          2m
 kube-system   oidc-gangway-7b7fbbdbdf-ll6l8           1/1       Running   0          2m
 ----
+
+==== Co-installing multiple versions of kubectl
+
+It's possible to install multiple versions of `kubectl` on the same machine. In
+a {productname} management machine, you will be able to co-install multiple
+versions of `kubectl`, and later mark which is the version to be used with
+`update-alternatives`.
+
+This can be useful, for example, when managing multiple Kubernetes clusters from
+the same {productname} management machine. That is, following {productname} 4.5,
+{productname} will be able to manage other (supported) Kubernetes versions than
+only 1.17.
+
+For example, in a system like this:
+
+[source,bash]
+----
+# kubectl version
+Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.4", GitCommit:"8d8aa39598534325ad77120c120a22b3a990b5ea", GitTreeState:"clean", BuildDate:"2020-03-17T12:00:00Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
+Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.4", GitCommit:"8d8aa39598534325ad77120c120a22b3a990b5ea", GitTreeState:"clean", BuildDate:"2020-03-17T12:00:00Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
+----
+
+You can install another version and use it right away with `update-alternatives`:
+
+[source,bash]
+----
+# zypper install kubernetes-1.18-client
+# update-alternatives --set kubectl /usr/bin/kubectl-1.18
+----
```

Comment:
I'm still stuck on checking that this actually works on a real machine, so take it with a grain of salt (I won't be around tomorrow nor next week so I wanted to submit this PR anyways)

Reply:
Two notes here:
First, we are not delivering a 1.17 client that will be co-installable with 1.18.  The 1.17 packaging was written without update-alternatives so isn't compatible with that functionality, so the 1.17 "kubernetes-client" package will be removed when "kuberenetes-1.18-client" is installed.
Second, by default the highest version number will be selected by default by update-alternatives when installed.  So in the scenario described, 1.18 would already be selected and you would not need to run the "update-alternatives --set kubectl /usr/bin/kubectl-1.18" command to use it.  If, after installing 1.19 then the user wanted to go back to 1.18, then they would use that command.

Reply:
I agree with @jodavis-suse . Next to this, I would prefer if we don't encourage people to install the client on the master/worker nodes, as this is handled by skuba. We don't want users to mess up the expected versions there. In order to clarify that this is for the management machine, I would mention `You can install another version *on a management node* and use it right away with `update-alternatives` on L823

Reply:
Fixed